### DISCUSSION TITLE
전체적인 분위기 레퍼런스

### DISCUSSION THREAD

**Discussion by sueyeoun** [The-Day-In-1984/1984_Project#71] [Q&A]
bgm 작업이 본격적으로 들어가기 전에 알맞은 레퍼런스랑 악기들을 서치하려 하는데 혹시 생각한 분위기가 있을까?
이미지로 제시해 주면 좋고 음악으로 알려주면 더 좋아!! 

@aminseo1023 @dkssu46

**Comment by dkssu46** [comment]
일단 나도 레퍼런스 음악 몇개 찾아보고 있을게!

**Comment by fkdl0048** [comment]
https://www.youtube.com/watch?v=HMXN8xClCrU&t=1s&ab_channel=Alaemortis

- this war of mine 분위기가 비슷..?

https://www.youtube.com/watch?v=ueE8G45cWDs&ab_channel=%EC%9B%94%ED%8A%B8%EA%B2%8C%EC%9E%84%EC%8A%A4%ED%86%A0%EB%A6%AC

- 리틀 나이트 메어인데 가장 끌리는..

이런 느낌..? 아트 풍이 귀여워서 좀 더 가벼워도 될 것 같은데..

**Reply by sueyeoun** [reply]
오 큰 틀에서 보면 전체적인 분위기는 크게 다르지 않다!! 고마워!! 신스로 분위기 조절하는 게 관건이겠구만~! 굿굿

**Comment by aminseo1023** [comment]
https://www.youtube.com/watch?v=-K0Ne8otETY
Stray인데 요런 느낌도 괜찮을 것 같아 더 찾아보고 끌리는 거 있으면 전달할게...!

**Reply by sueyeoun** [reply]
스트레이.. 나 너무 좋아하는데 이것도 분위기 조성이 관건이겠당 고마워~!!!
